src/main/java/com/mealmentor/enterprise/dao/MealItemDAOStub.java

```diff
@@ -13,16 +13,15 @@
     HashMap<Integer, MealItem> allMealItems = new HashMap<>();
 
     @Override
-    public MealItem save(MealItem mealItem) throws Exception {
+    public MealItem save(MealItem mealItem) {
```

Comment:
I saw this several times in the code. Method signatures say they throw an exception but then never throw an exception.